src/app/components/display-request-page/display-request-page.component.ts

```diff
@@ -93,6 +95,24 @@ export class DisplayRequestPageComponent implements OnInit {
   }
   }
 
+  //Toggles the select format strings
+toggleFormat(format : string){
+
+
+      
+  if(this.selFormat.includes(format)){
+     if(this.selFormat.length==1){
```

Comment:
Same question as previously, should this if statement be this.selFormat.length == 0? Unless I am missing something.

Reply:
Please see previous response.